libpod/events/logfile.go

```diff
@@ -29,7 +29,7 @@ type EventLogFile struct {
 // newLogFileEventer creates a new EventLogFile eventer
 func newLogFileEventer(options EventerOptions) (*EventLogFile, error) {
 	// Create events log dir
-	if err := os.MkdirAll(filepath.Dir(options.LogFilePath), 0700); err != nil {
+	if err := os.MkdirAll(filepath.Dir(options.LogFilePath), 0600); err != nil {
```

Comment:
Why 600?  Is this a directory you don't want the owner to search through?

Reply:
You are ceating a directory here, not a file, so the commit message and the code change do not match.